packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -535,13 +552,19 @@ export enum RuntimeMessage {
 }
 
 export function isRuntimeMessage(message: ISequencedDocumentMessage): boolean {
-    if ((Object.values(RuntimeMessage) as string[]).includes(message.type)) {
-        return true;
-    }
-    return false;
+    return (Object.values(RuntimeMessage) as string[]).includes(message.type);
 }
 
 export function unpackRuntimeMessage(message: ISequencedDocumentMessage) {
+    if (message.metadata?.compressed) {
+        const contents = IsoBuffer.from(message.contents.contents, "base64");
+        const decompressedMessage = decompress(contents);
+        const intoString = new TextDecoder().decode(decompressedMessage);
+        const asObj = JSON.parse(intoString);
```

Comment:
One thing to point out (that I think I pointed out before): DeltaManager does extra stringification, which we do not want long term. So ideally, we do not make any promises yet about supporting it long term in the form it is now. For that, I believe we have some tag (like @experimental?) that should probably be on IContainerRuntimeOptions.compressionOptions option.

Reply:
Added `@experimental` tag to the `compressionOptions` option.